streaming/src/main/scala/com/stratio/crossdata/streaming/helpers/CrossdataStreamingHelper.scala

```diff
@@ -38,6 +38,9 @@ object CrossdataStreamingHelper extends SparkLoggerComponent {
                     crossdataCatalogConf: Map[String, String]): StreamingContext = {
     val sparkStreamingWindow = ephemeralTable.options.atomicWindow
     val sparkContext = SparkContext.getOrCreate(sparkConf)
+
+    val countdowns = collection.mutable.Map[String, Int]()
```

Comment:
Would it be too bizarre to use Spark accumulators instead of a map?
http://spark.apache.org/docs/latest/programming-guide.html#accumulators